useful-scripts/misc-test-utils/src/modules/publication/createPublication.ts

```diff
@@ -1,12 +1,13 @@
 /* eslint-disable no-console */
 import chalk from 'chalk';
+import logger from '../../utils/logger';
 import publicationService from '../../services/publicationService';
 
 const { ADMIN_URL } = process.env;
 
 const createPublication = async () => {
   const publicationId = await publicationService.createPublication();
-  console.log(
+  logger.info(
```

Comment:
Nice to have this - we should add this to the perf tests when we get time too!

Reply:
Yeah it's a pretty nice logging library